src/mower_logic/src/mower_logic/behaviors/MowingBehavior.cpp

```diff
@@ -143,6 +143,11 @@ bool MowingBehavior::create_mowing_plan(int area_index) {
         }
     }
 
+    // use Area orientation from config if not default value
+    if (config.angle != 0) {
+        angle = (config.angle * (M_PI / 180.0));
+        ROS_INFO_STREAM("MowingBehavior: Config mow angle: " << angle);
+    }
```

Comment:
I think 0 should be a valid mowing angle as well. We should probably have something like `config.has_custom_mow_angle` which gets set to true, if OM_MOWING_ANGLE is set, false otherwise

Reply:
How about rename the param to `mow_angle_offset` and add a `mow_angle_offset_is_absolute` boolean? If `true` then the offset is relative to East, if `false` then the offset is relative to the auto-detected angle.

Reply:
yes sounds good to me